global.json

```diff
@@ -757,7 +757,7 @@
       "appTemplates": ["hello_world_mbaas_instance"]
     }, {
       "name": "OpenShift mBaaS Service",
-      "priority": 0.895,
```

Comment:
Why we changing priority of this? IMHO That service should be removed as it's used only for openshift2 which is no longer working. Only one way to get openshift2 clusters was do use openshift.feedhenry.net but I think we disabled that for the moment as there is no way to register new accounts to OpenShift .